backend/django_core/static_src/components/minor/Comment/CommentBlock.svelte

```diff
@@ -100,9 +98,17 @@
         }
     };
 
-    const handle_more_click = async (path: string) => {
-        dispatch("more_comments", {
-            path: path,
+    const handle_more_click = async () => {
+        const comment_path = item.path;
+        const comment_api_url = `/api/v2/comments/?path=${comment_path}`;
+
+        fetch_comments(comment_api_url).then((res) => {
+            const new_comment = new JSONToTree({
+                json: res.results,
+                specific_path: comment_path,
+            }).build() as unknown as Comment[];
```

Comment:
You should not add just one comments. The endpoint returns a lot of comments. Its due to https cost.

Reply:
> The endpoint returns a lot of comments.

It returns children of that specific comment path.
Then process to a tree?

at the end its just one comment and its children, no?